common4j/src/main/com/microsoft/identity/common/java/net/HttpRequest.java

```diff
@@ -42,10 +43,17 @@
     @Accessors(prefix = "m")
     private final URL mRequestUrl;
 
-    @Getter
     @Accessors(prefix = "m")
     private final byte[] mRequestContent;
 
+    public byte[] getRequestContent() {
+        if (mRequestContent == null) {
+            return null;
+        }
+        
+        return Arrays.copyOf(mRequestContent, mRequestMethod.length());
```

Comment:
This is probably breaking your tests

```suggestion
        return Arrays.copyOf(mRequestContent, mRequestContent.length);
```